### PR TITLE
Some clean up

### PR DESCRIPTION
**Pull request by ammsalme**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -45,68 +48,82 @@ export default async function ({
...
-      logger.silly(`app/checkItemImportingAndInQueue: Nothing found: PRIO -> checkItemImportingAndInQueue `);
-      return checkItemImportingAndInQueue(false);
+      logger.silly(`app/checkItemImportingAndInQueue: Nothing found: PRIO -> checkItemImportingAndInQueue for bulk`);
+      return checkItemImportingAndInQueue({prio: false, waitSinceLastOp});
     }
 
-    logger.silly(`app/checkItemImportingAndInQueue: Nothing found: BULK -> startCheck `);
-    return startCheck(true, 3000);
+    logger.silly(`app/checkItemImportingAndInQueue: Nothing found: BULK -> startCheck, waiting ${pollWaitTime} ms = ${pollWaitTime / 1000} s`);
+    return startCheck({checkInProcess: true, wait: pollWaitTime, waitSinceLastOp});
```
Comment (ammsalme) [inline src/app.js:106]: This fixes hard coded waitTime of 3000 ms to waitTime configured in the environment variable POLL_WAIT_TIME

File: src/app.js [hunk 2]
```diff
@@ -146,30 +170,44 @@ export default async function ({
       }
 
       if (queueItem && queueItem.importJobState[operation] === IMPORT_JOB_STATE.IN_QUEUE) {
-        logger.debug(`Found item in queue to be imported ${queueItem.correlationId}`);
+        logger.debug(`Found item in q ueue to be imported${queueItem.correlationId} ${waitTimePrint(waitSinceLastOp)}`);
```
Comment (natlibfi-jonollil) [inline src/app.js:173]: typo

File: src/app.js [hunk 3]
```diff
@@ -146,30 +170,44 @@ export default async function ({
...
 
+        logger.debug(`Found item in queue to be imported ${queueItem.correlationId} ${waitTimePrint(waitSinceLastOp)}`);
         await mongoOperator.setState({correlationId: queueItem.correlationId, state: QUEUE_ITEM_STATE.IMPORTER.IMPORTING});
         return true;
       }
       return false;
     }
   }
+
+  // We should move this to some commons
```
Comment (natlibfi-jonollil) [inline src/app.js:201]: Lets move it is melinda-backend-commons
and change it to print once in 15mins (900000ms)

**Review by natlibfi-jonollil — CHANGES_REQUESTED** [review]

File: src/app.js (hunk 2 above)
Comment (ammsalme) [reply src/app.js]: Fixed

File: src/app.js (hunk 3 above)
Comment (ammsalme) [reply src/app.js]: Moved to melinda-backend-commons: https://github.com/NatLibFi/melinda-backend-commons-js/pull/21 #

**Pull request merged**